pennylane/transforms/condition.py

```diff
@@ -73,13 +94,17 @@ def cond(condition, true_fn, false_fn=None):
             apply if ``condition`` is ``True``
         false_fn (callable): The quantum function of PennyLane operation to
             apply if ``condition`` is ``False``
+        elifs (List(Tuple(bool, callable))): A list of (bool, elif_fn) clauses only when
+            they used with :func:`~.qjit`
 
     Returns:
         function: A new function that applies the conditional equivalent of ``true_fn``. The returned
         function takes the same input arguments as ``true_fn``.

```

Comment:
Does this need updating?

Reply:
Good question! I didn't update the return type as this doesn't make sense when is called inside a QJIT-ed workflow. Should I update this maybe as follows:
```suggestion
    Returns:
        function: In interpreted mode, this returns a new function that applies the conditional equivalent
         of ``true_fn``. The returned function takes the same input arguments as ``true_fn``.
```